### PR TITLE
Update config.yaml

### PR DESCRIPTION
**Pull request by ashokjammula1**
New member of babylon and RHPDS team

### CODE REVIEW

**Comment by 9strands** [comment]
Hi, Redhat-cop maintainers, I confirm that @ashokjammula1 is part of my team and will require the same access that I have.  Please approve so that we can get @ashokjammula1 into our workflows and training.  Thank you!

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Thank you @ashokjammula1 and thank you @9strands for the confirmation.
This looks good but could you please update the PR to keep the usernames in alphabetical order?

**Comment by 9strands** [comment]
Hi, @pabrahamsson - yes, I will work with @ashokjammula1 to update and keep alpha order.

**Comment by 9strands** [comment]
@pabrahamsson - updated, I believe this is as you requested.

File: config.yaml [hunk 1]
```diff
@@ -1085,6 +1086,7 @@ orgs:
           - wkulhanek
         members:
           - 9strands
+          - ashokjammula1
           - ahsen-shah
           - aleixhub
```
Comment (pabrahamsson) [inline config.yaml:1091]:
```suggestion
          - ahsen-shah
          - aleixhub
          - ashokjammula1
```

File: config.yaml [hunk 2]
```diff
@@ -1147,6 +1150,7 @@ orgs:
                 maintainers:
                   - 9strands
                   - ahsen-shah
+                  - ashokjammula1
                   - aleixhub
```
Comment (pabrahamsson) [inline config.yaml:1154]:
```suggestion
                  - aleixhub
                  - ashokjammula1
```

**Review by pabrahamsson — COMMENTED** [review]
Thank you for the update, I left suggested changes inline.

**Review by pabrahamsson — APPROVED** [review]
LGTM

**Pull request merged**